src/status.rs

```diff
@@ -386,7 +394,7 @@ impl Deserialize for Status {
                         }
                     }
                 }
-                visitor.end()?;
+                //visitor.end()?;
```

Comment:
According to https://github.com/serde-rs/serde/releases/tag/v0.9.0 it isn't needed anymore.